tests/mockduo.py

```diff
@@ -119,9 +124,12 @@ def _get_tx_response(self, txid, is_async):
         time.sleep(int(secs))
         return rsp
 
-    def _send(self, code, buf=b""):
+    def _send(self, code, buf=b"", headers={}):
```

Comment:
It's a convention in python to avoid using an empty dict (or list) as an argument default.  There are use cases where it doesn't behave as you'd expect (https://stackoverflow.com/questions/26320899/why-is-the-empty-dictionary-a-dangerous-default-value-in-python for example).